packages/web-client/app/components/card-pay/merchant-payment-request-page/index.hbs

```diff
@@ -0,0 +1,70 @@
+{{!--
+  TODO:
+  - replace hardcoded merchant info with properties of the component class
+  - update media queries of this and merchant-payment-request-card to use 480px (conventional upper limit for smartphone)
+  - update icon/logo size for responsive design, +- updating org title for better control of spacing betweem logo and text
+ --}}
+
+<div class="merchant-payment-request-page">
+  <div class="merchant-payment-request-page__header">
+    <Boxel::OrgTitle
+      @title="Card Pay"
+      @iconURL="/images/icons/card-pay-logo.svg"
+      style={{css-var
+        boxel-org-title-logo-size="var(--card-pay-logo-size)"
+        boxel-org-title-logo-position="left center"
+      }}
+    />
+
+    <div class="merchant-payment-request-page__links-container">
+      <ul class="merchant-payment-request-page__links">
+        <li>
+          <a href="#">
+            About
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            Help
+          </a>
+        </li>
+      </ul>
+      <CardPay::HamburgerMenu
+        class="merchant-payment-request-page__hamburger"
+        @open={{this.hamburgerMenuOpen}}
+        @onOpen={{fn (mut this.hamburgerMenuOpen) true}}
+        @onClose={{fn (mut this.hamburgerMenuOpen) false}}
+      >
+        <:header>
+          <Boxel::OrgTitle
+            @title="Card Pay"
+            @iconURL="/images/icons/card-pay-logo.svg"
+            style={{css-var
+              boxel-org-title-logo-size="var(--card-pay-logo-size)"
+              boxel-org-title-logo-position="left center"
+            }}
+          />
+        </:header>
+        <:body as |MenuItem|>
+          <MenuItem @url="#">
+            About
+          </MenuItem>
+          <MenuItem @url="#">
+            Help
+          </MenuItem>
+        </:body>
+      </CardPay::HamburgerMenu>
```

Comment:
This is where and how the hamburger menu is used

Reply:
We are controlling the visibility of the hamburger menu via media query at the moment, but toggling whether it's open or not via Javascript. It's possible to set `this.hamburgerMenuOpen` to `false` via a JS media query(`window.matchMedia`), but unsure if necessary.